iron-doc-viewer.html

```diff
@@ -56,7 +57,10 @@
 
     <nav id="api">
       <header>API Reference</header>
-      <a id="togglePrivate" on-tap="_togglePrivate">{{_privateToggleLabel}}</a>
+      <paper-button id="togglePrivate"
+        tabindex="0"
+        role="button"
```

Comment:
paper-button will automatically get role=button if you don't specify a role, so you can leave this off.